bin/pre-push.ts

```diff
@@ -55,7 +56,8 @@ if (refs?.[0]?.localCommit.match(/^0+$/)) {
   process.exit(0)
 }
 
-const packageVersion = require('../package.json').version
+const pkgFile = path.join(process.cwd(), 'package.json')
```

Comment:
the `process.cwd()` will be depended on the Current Working Directory (CWD), which is not reliable.

It would be better to use the `__dirname` so that the path can be absolute.